### PR TITLE
deps: update feelin to version 1.0.0

### LINKED ISSUE #16: Update to `feelin@1`
### What should we do?

`lezer-feel` recieved a major update (cf. [Changelog](https://github.com/nikku/lezer-feel/blob/master/CHANGELOG.md#100)). We should carefully investigate what this means for our FEEL support and update it accordingly.

* [ ] Bump to `feelin@1`
* [ ] Investigate 
* [ ] Mitigate breaking changes (if needed)

### Why should we do it?

Keep up to date with our FEEL foundations.

### PR DESCRIPTION
**Pull request by Skaiir**
Closes #16

### CODE REVIEW

File: test/spec/singleStart.spec.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe('CodeEditor', function() {
   });
 
 
-  (singleStart ? it.only : it)('should render', async function() {
+  (singleStart ? it.only : it.skip)('should render', async function() {
```
Comment (pinussilvestrus) [inline test/spec/singleStart.spec.js:19]: This skip needs to be removed, right?

**Review by pinussilvestrus — COMMENTED** [review]
Dry run in form-js looks good 👍 

After merging this, let's release a stable version, I think alpha stage can be left now 🚀

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Cache Node.js modules
```
Comment (nikku) [inline .github/workflows/CI.yml:20]: You can drop cache completely in favor of built in caching.

File: .github/workflows/CI.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
```
Comment (nikku) [inline .github/workflows/CI.yml:19]:
```suggestion
        node-version: ${{ matrix.node-version }}
        cache: 'npm'
```

File: test/spec/singleStart.spec.js (hunk 1 above)
Comment (Skaiir) [reply test/spec/singleStart.spec.js:19]: We already have a render test, I skipped this one as we do an evaluation through console.log and I didn't want it to affect the logs, but in singleStart it's nice to see it in the browser logs at least.

**Review by pinussilvestrus — APPROVED** [review]
💯

**Pull request merged**